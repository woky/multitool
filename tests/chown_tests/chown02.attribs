0 0 files
0 0 files/A
0 0 files/A/B
0 0 files/A/x
0 0 files/A/y
9 0 files/B
9 0 files/B/A
9 0 files/B/B
9 0 files/B/B/A
9 0 files/B/B/x
9 0 files/B/B/y
9 0 files/B/ax
9 0 files/B/x
9 0 files/B/y
