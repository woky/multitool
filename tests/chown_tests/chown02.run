#!/bin/sh
/proj/multitool chown -R news files/B
