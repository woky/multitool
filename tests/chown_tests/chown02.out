files/B/A
files/B/B/A
files/B/B/x
files/B/B/y
files/B/B
files/B/ax
files/B/x
files/B/y
files/B
